db/migrate/20200310200000_remove_timezone_custom_field.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveTimezoneCustomField < ActiveRecord::Migration[5.2]
+  def change
+    UserCustomField.where(name: 'timezone').delete_all
```

Comment:
Can you use SQL here instead? We've had issues historically with objects in migrations.

Reply:
Of course, I will do this.